core/templates/pages/learner-dashboard-page/new-learner-dashboard/community-lessons-tab.component.html

```diff
@@ -0,0 +1,654 @@
+<div class="community-lessons-section">
+  <p class="lessons-title" [innerHTML]="'I18N_LEARNER_DASHBOARD_COMMUNITY_LESSONS_SECTION' | translate">
+  </p>
+  <div class="dropdown" *ngIf="checkMobileView() && (totalCompletedLessonsList.length !== 0 || totalIncompleteLessonsList.length !== 0)">
+    <span class="view-text">
+      View
+    </span>
+    <button (click)="enableDropdown()" class="dropbtn">
+      {{selectedSection}}
+      <i class="fa fa-caret-down dropdown-icon"></i>
+    </button>
+    <div class="dropdown-content" *ngIf="dropdownEnabled">
+      <p (click)="changeSection('All')">All</p>
+      <p (click)="changeSection('Completed')">Completed</p>
+      <p (click)="changeSection('Incomplete')">Incomplete</p>
+    </div>
+    <div class="pagination" *ngIf="displayInCommunityLessons.length !== 0">
+      <i class="fa fa-angle-left pagination-arrow" aria-label="Previous Page" (click)="changePageByOne('MOVE_TO_PREV_PAGE', 'communityLessons')"></i>
+      <span class="dot">
+        <span class="pagination-number">
+          {{pageNumberInCommunityLessons}}
+        </span>
+      </span>
+      <i class="fa fa-angle-right pagination-arrow" aria-label="Next Page" (click)="changePageByOne('MOVE_TO_NEXT_PAGE', 'communityLessons')"></i>
+    </div>
+  </div>
+  <div *ngIf="noCommunityLessonActivity">
+    <p [innerHTML]="'I18N_LEARNER_DASHBOARD_NO_ACTIVITY_IN_EXPLORATION' | translate" class="empty-section"></p>
```

Comment:
I just found a typo in the text for this translation in en.json. It says 'explanation' instead of 'exploration'. Can you fix that?

Reply:
Done.